frontend/src/routes/manage/Shared/Details.tsx

```diff
@@ -200,18 +202,20 @@ export const MetadataSection = <TMutation extends MetadataMutationParams>({
         });
     });
 
-    return <Inertable isInert={!isSynced(item)}>
+    return <Inertable isInert={disabled || !isSynced(item)}>
         <FormProvider {...formMethods}>
             <MetadataForm hasError={!!commitError}>
                 {/* Title & Description */}
-                <MetadataFields disabled={!commit} />
+                <MetadataFields />
```

Comment:
Shouldn't `disabled` be forwarded to `MetadataFields`?

Reply:
It doesn't need to be since this is wrapped in the `<Inertable>` component, which disables everything inside it when the specified condition is met. On the other hand it wouldn't hurt either.